frontend/src/components/SearchTable.vue

```diff
@@ -633,7 +632,7 @@ export default {
     next();
   },
   updated() {
-    $('#search').focus();
+    document.getElementById('search').focus();
```

Comment:
Maybe silly question,  but what functionality does this line contribute with?

Reply:
This focuses the search input element each time after a search is performed. I assume it's convenient for people who are performing multiple searches. Personally I don't know how often this is needed but at the same time I don't think it hurts anything to have it.